media/css/firefox/accounts-2019.scss

```diff
@@ -25,8 +25,6 @@
 
     .c-accounts-hero-title {
         @include text-display-lg;
-        background: transparent url('/media/protocol/img/logos/firefox/logo-word-hor.svg') center top no-repeat;
```

Comment:
This seems unrelated to this issue, but if this is for https://github.com/mozilla/bedrock/issues/8118 then let's remove the logo from exp/firefox/accounts/ to?

Reply:
oh whoops, this was accidentally left in while I was filing that issue. 🤦‍♀